src/LoRa.h

```diff
@@ -57,9 +57,11 @@ class LoRaClass : public Stream {
 
 #ifndef ARDUINO_SAMD_MKRWAN1300
   void onReceive(void(*callback)(int));
+  void onCadDone(void(*callback)(boolean));
   void onTxDone(void(*callback)());
 
   void receive(int size = 0);
+  void CAD(void);
```

Comment:
Thoughts on making this lower case or expanded to something like `collisionActivityDetect()`?

Reply:
Doesn't it stand for Channel Activity Detection? Additionally, I thought the other day this should be named to make it clear this is CAD *DONE*, as there is also `CadDetect`

Reply:
These methods are well self-descriptive enough IMHO. channelActivityDetection switch the receiver to CAD mode, onCadDone register a callback function for CADDone interrupts.
````arduino
void channelActivityDetection(void);
void onCadDone(void(*callback)(boolean));
````